docs/build/schema.md

```diff
@@ -195,6 +196,31 @@ Key | Required | Type | Description
 
 -----
 
+## /BasicActionOperationSchema
+
+Represents the fundamental mechanics of triggers, searches, or creates.
+
+#### Details
+
+* **Type** - `object`
+* **Pattern** - _n/a_
+* **Source Code** - [lib/schemas/BasicActionOperationSchema.js](https://github.com/zapier/zapier-platform-schema/blob/v0.9.12/lib/schemas/BasicActionOperationSchema.js)
+
+
+
+#### Properties
+
+Key | Required | Type | Description
+--- | -------- | ---- | -----------
+`resource` | no | [/RefResourceSchema](#refresourceschema) | Optionally reference and extends a resource. Allows us to automatically tie together samples, lists and hooks greatly improving the UX. EG: if you had another trigger reusing a resource but filtering the results.
+`perform` | **yes** | oneOf([/RequestSchema](#requestschema), [/FunctionSchema](#functionschema)) | How will we get the data? This can be a function like `(z) => [{id: 123}]` or a request like `{url: 'http...'}`.
+`performGet` | no | oneOf([/RequestSchema](#requestschema), [/FunctionSchema](#functionschema)) | How will we get a single bit of rich data? If you find yourself reaching for this - consider resources and their built-in get methods.
+`inputFields` | no | [/DynamicFieldsSchema](#dynamicfieldsschema) | What should the form a user sees and configures look like?
+`outputFields` | no | [/DynamicFieldsSchema](#dynamicfieldsschema) | What fields of data will this return? Will use resource outputFields if missing, will also use sample if available.
+`sample` | no | `object` | What does a sample of data look like? Will use resource sample if missing.
```

Comment:
This sample is the only one for the operation, and we want try to do a performGet to fetch more data, so I think this is correct.